src/iomarket.cpp

```diff
@@ -158,23 +156,21 @@ void processExpiredOffers(DBResult_ptr result, bool)
 				}
 
 				for (uint16_t i = 0; i < amount; ++i) {
-					Item* item = Item::CreateItem(itemType.id, subType);
-					if (g_game.internalAddItem(player->getInbox().get(), item, INDEX_WHEREEVER, FLAG_NOLIMIT) !=
+					auto item = Item::CreateItem(itemType.id, subType);
```

Comment:
Could this be `const auto`
```suggestion
					const auto item = Item::CreateItem(itemType.id, subType);
```